tests/filters/test_monster_filter.py

```diff
@@ -41,6 +41,31 @@ def test_monster_id(self):
         for e in [fail1, fail2, fail3]:
             self.assertFalse(mon_filter.check_event(e))
 
+    def test_ignore_monster_id(self):
+        # Create the filters
+        settings = {
+            "monsters": [1, "2", "Venusaur"],
```

Comment:
For testing here, I would remove `"monsters"` as we truly just want to test the ignore filter in this particular scenario.

Reply:
sure thing, updated